application/src/test/java/org/opentripplanner/street/model/edge/StreetEdgeGeofencingTest.java

```diff
@@ -255,7 +255,7 @@ public void pickupFloatingVehiclesWhenStartedInNoDropOffZone() {
       assertEquals(WALK, walkState.currentMode());
 
       // then the speculative renting case for unknown rental network
-      final State speculativeRenting = states[1];
+      final State speculativeRenting = states[2];
```

Comment:
Why did the order of states change btw?

Reply:
We only know if the unknown network case will be generated after generating the known cases. With recursive streams it was stuffed in the middle, but with an array it's easier (and more performant) to put it in the end. It shouldn't really matter what the order is, but in debugging it can be helpful that it is stable, so using State[] and not Set<State> is understandable.